px4_ros2_cpp/src/navigation/experimental/global_navigation_interface.cpp

```diff
@@ -0,0 +1,131 @@
+/****************************************************************************
+ * Copyright (c) 2023 PX4 Development Team.
+ * SPDX-License-Identifier: BSD-3-Clause
+ ****************************************************************************/
+
+#include <px4_ros2/navigation/experimental/global_navigation_interface.hpp>
+
+
+namespace px4_ros2
+{
+
+GlobalNavigationInterface::GlobalNavigationInterface(rclcpp::Node & node)
+: NavigationInterfaceBase(node)
+{
+  _aux_global_position_pub =
+    node.create_publisher<VehicleGlobalPosition>(
+    topicNamespacePrefix() + "/fmu/in/aux_global_position", 10);
+}
+
+NavigationInterfaceReturnCode GlobalNavigationInterface::update(
+  const GlobalPositionEstimate & global_position_estimate) const
+{
+  // Run basic sanity checks on global position estimate
```

Comment:
Do you mean in the PX4 user guide? Each check has a `@brief` description, but how do you mean otherwise?

Reply:
> Ah or do you mean to extend the @brief description of the update method?

I'd do this, to have it in the code. But mentioning it in the user guide is also a good idea

Reply:
Exactly, for the `update`, since that is what someone using the API most likely will look at (you could also link to individual checks if there's more text to avoid repetition)